code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1164,6 +1164,26 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		. = 1
 	return ..() || .
 
+/datum/reagent/consumable/ethanol/beuro
+	name = "Bureaucratic Perfection"
+	description = "Smells of freshly signed and filed papers.."
+	nutriment_factor = 1 * REAGENTS_METABOLISM
+	color = "#806029" // rgb: 128, 86, 41 (what is this used for?)
+	boozepwr = 20
+	quality = DRINK_GOOD
+	taste_description = "pure organization"
+	glass_icon_state = "beuroglass"
+	glass_name = "Bureaucratic Perfection"
+	glass_desc = "Smells of freshly signed and filed papers.."
+	name = "Bureaucratic Perfection"
+	description = "Smells of freshly signed and filed papers.."
+
+/datum/reagent/consumable/ethanol/beuro/on_mob_life(mob/living/carbon/M)  // this is 100% copy pasted but im new to this
+	if((ishuman(M) && (M.job == "Cargo Technician" || M.job == "Quartermaster" || M.job == "Head of Personnel")) || ismonkey(M))  // heals jobs relating to paperwork :)
```

Comment:
Why does it heal if you're a monkey?

Reply:
Copied code from the banana drink right above it